windows/src/global/delphi/cust/CustomisationMessages.pas

```diff
@@ -107,61 +107,6 @@ function TCustomisationMessageManager.GetAvailableLanguages: string;
     end;
   end;
 
-  procedure ReadPackageLanguages(IsLM: Boolean);
```

Comment:
Noting the message from this commit:

> Locales will be distributed as part of the app, for now. We may re-introduce package-based locales in the future, across all platforms.

Would this affect users in any noticeable way or cause possible loss of functionality for people?

Reply:
There may be some very old packages that include locale.xml files, but it's never been very well supported and I have never seen any in the wild. As it stands, they won't work anyway since we went from locale.xml format to strings.xml format for 14.0...